fury/actor.py

```diff
@@ -1270,3 +1272,145 @@ def marker(
 
     obj = create_point(geo, mat)
     return obj
+
+
+def slicer(
+    data,
+    *,
+    value_range=None,
+    opacity=1.0,
+    interpolation="linear",
+    visibility=(True, True, True),
+    initial_slices=None,
+):
+    """Visualize a 3D volume data as a slice.
+
+    Parameters
+    ----------
+    data : ndarray, shape (X, Y, Z) or (X, Y, Z, 3)
```

Comment:
do you plan to create slicer for 4D data later on ? (e.g diffusion data)

Reply:
The current slicer always works with 3D data only. In Horizon, we create a new slicer for each volume. As we should not load and create entire data.

If we want to do it in FURY I can incorporate that in medical actor for the slicer.

Reply:
ok sounds good. thanks for clarification !